### PR TITLE
com.darhon.drfinance

### PR DESCRIPTION
**Pull request by DarhonSoftware**
### Please confirm your submission meets all the criteria

- [ ] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [ ] My pull request follows the instructions at [App Submission][submission].
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [ ] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.darhon.drfinance.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: drfinance
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=host
```
Comment (hfiguiere) [inline com.darhon.drfinance.yml:11]: why `host`

**Comment by hfiguiere** [comment]
bot, build com.darhon.drfinance

File: com.darhon.drfinance.yml (hunk 1 above)
Comment (DarhonSoftware) [reply com.darhon.drfinance.yml:11]: It is mainly to allow users to access an external drive (USB).

**Comment by DarhonSoftware** [comment]
bot, build com.darhon.drfinance

**Pull request closed without merging**